test/net/sourceforge/kolmafia/session/OysterEggTest.java

```diff
@@ -0,0 +1,29 @@
+package net.sourceforge.kolmafia.session;
+
+import static internal.helpers.Player.isDay;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.GregorianCalendar;
+import net.sourceforge.kolmafia.AdventureResult;
+import net.sourceforge.kolmafia.KoLCharacter;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+
+public class OysterEggTest {
+  public void OysterEggObtainingTest() {
+    KoLCharacter.reset("the Tristero");
```

Comment:
Instead of this, add a `@BeforeEach` to the class that runs `KoLCharacter.reset(<className>)` and Preferences.reset(<className>)`. I started using the class name for a username because it limits any unexpected statefullness to that single class.